src/main/groovy/com/blackducksoftware/integration/hub/artifactory/inspect/blackDuckCacheInspector.groovy

```diff
@@ -39,7 +82,7 @@ private void populateHubOriginId(RepoPath repoPath) {
     ExternalId externalId = constructExternalIdFromRepoPath(repoPath)
     if (externalId) {
         String hubOriginId = externalId.createHubOriginId()
-        repositories.setProperty(repoPath, 'blackduck.hubOriginId', externalId.createHubOriginId())
+        repositories.setProperty(repoPath, 'blackduck.hub.origin.id', externalId.createHubOriginId())
```

Comment:
We should keep the lunatic convention, since it seems to be the artifactory standard